lighthouse-core/report/v2/report-styles.css

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ :root {
+  --text-font-family: '.SFNSDisplay-Regular', 'Helvetica Neue', 'Lucida Grande', sans-serif;
+  --body-font-size: 13px;
+
+  --secondary-text-color: #565656;
```

Comment:
+1. I've been operating under the assumption that our baseline is the latest version of each modern browser.

Good news is [everyone has CSS custom props](http://caniuse.com/#feat=css-variables) except for Edge. And they'll get them in their next release (Edge 15).

Regardless, Edge will be an investment that we haven't made before and it might not be worth it. They're catching up on features quickly. Any work we put in now might be moot in the near future. FWIW, < 0.60% Viewer users are using Edge...so a fraction of the already small browser share :) I'd be curious to see the analytics for WPT though.

One thing that would also be interesting is offloading polyfills and broader browser support to Viewer. The more code we to ship in the binary, the larger the report output becomes.